axelor-base/src/main/resources/views/GenerateMessageWizard.xml

```diff
@@ -3,6 +3,26 @@
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://axelor.com/xml/ns/object-views http://axelor.com/xml/ns/object-views/object-views_6.1.xsd">
 
+  <form name="generate-message-wizard-form" model="com.axelor.utils.db.Wizard"
+    id="base-generate-message-wizard-form" title="Generate message"
+    onNew="action-generate-message-group-on-new" extension="true">
+
+    <extend target="//panel[@name='mainPanel']/field[@name='language']">
+      <attribute name="name" value="localization"/>
+      <attribute name="title" value="Localization"/>
+      <attribute name="target" value="com.axelor.apps.base.db.Localization"/>
+      <attribute name="onChange" value="action-on-change-localization-template"/>
```

Comment:
no just use replace tag and put directly the field localization, it's weird to do it that way